modules/series-service-remote/src/main/java/org/opencastproject/series/remote/SeriesServiceRemoteImpl.java

```diff
@@ -118,6 +118,9 @@ public SeriesServiceRemoteImpl() {
   /** Default number of items on page */
   private static final int DEFAULT_LIMIT = 20;
 
+  /** Default character encoding */
+  private static final String ENCODING = "UTF-8";
```

Comment:
You did replace some, but not all occurrences of "UTF-8" in the code. If we introduce this, it probably makes sense to replace them all. Also, would it make sense to use `java.nio.charset.StandardCharsets.UTF_8` instead?

Reply:
I don't know what happened but IntelliJ had troubles to find the other UTF-8 occurrences. It definitely makes sens to use the StandardCharsets.UTF_8 instead.
.